go.mod

```diff
@@ -231,3 +232,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/go-chef/chef v0.28.0 => github.com/chef/go-chef v0.4.5
```

Comment:
why are we using a replace, why don't we just depend on the other version instead?